content/blog/newsletter/2024-10-05-newsletter.md

```diff
@@ -0,0 +1,195 @@
+- [Psi+ 1.5.2041 installer](https://sourceforge.net/projects/psiplus/files/Windows/Personal-Builds/KukuRuzo/) has been released.
+- [Gajim 1.9.4](https://gajim.org/post/2024-09-19-gajim-1.9.4-released/) and [1.9.5](https://gajim.org/post/2024-09-30-gajim-1.9.5-released/) have been released. These releases come with integrated support for the [XMPP Providers](https://providers.xmpp.net/) project. Furthermore, there is now suppport for "Hats" ([XEP-0317](https://xmpp.org/extensions/xep-0317.html)), which allow you to assign roles to group chat participants, i.e. "Support", "Expert" or really anything you like to assign. Last but not least, Gajim’s Microsoft Store release has been improved in many ways. You can check the [changelog](https://dev.gajim.org/gajim/gajim/-/blob/master/ChangeLog) for more details.
+- [Movim 0.28](https://github.com/movim/movim/releases/tag/v0.28) has been released. This new version (code named "Tempel") brings a "Freshly redesigned Search panel, improved account gateways and administration features, databases fixes and a new call flow and conference lobby" among [many other fixes and improvements](https://mov.im/node/pubsub.movim.eu/Movim/movim-0-28-tempel-and-the-movim-live-plan-4EO29q).
+
+{{< figure src="/images/newsletter/2024-10-05/movim-0.28.png" caption="Movim 0.28 (Tempel) Introducing the new call flow and conference lobby" >}}
+
+### XMPP Servers
+
+- [Snikket](https://snikket.org/) is pleased to announce a new stable release of the Snikket server [stable.20240904](https://snikket.org/updates/release/20240904/) software. For an introduction to the new changes, check out the [release announcement blog post](https://snikket.org/blog/snikket-server-sept-2024-release/).
+- [Openfire 4.9.0 release!](https://discourse.igniterealtime.org/t/openfire-4-9-0-release/94688) The [Ignite Realtime community](https://www.igniterealtime.org/) is happy to announce the immediate availability of version [4.9.0](https://github.com/igniterealtime/Openfire/releases/tag/v4.9.0) of [Openfire](https://www.igniterealtime.org/projects/openfire/), its cross-platform real-time collaboration server based on the XMPP protocol! You can read the full [changelog](https://download.igniterealtime.org/openfire/docs/latest/changelog.html) for more details.
+
+### XMPP Libraries & Tools
+
+- [Hazelcast plugin for Openfire](https://www.igniterealtime.org/projects/openfire/plugin-archive.jsp?plugin=hazelcast) version [3.0.0](https://www.igniterealtime.org/projects/openfire/plugin-archive.jsp?plugin=hazelcast) has been released, adding clustering support to [Openfire](https://www.igniterealtime.org/projects/openfire/). It is based on [the Hazelcast platform](https://hazelcast.com/). This release brings a major upgrade of the platform. It migrates from version 3.12.5 to 5.3.7. Please [read the upgrade instructions carefully](https://www.igniterealtime.org/projects/openfire/plugins/3.0.0/hazelcast/readme.html) for this one.
```

Comment:
Usually we collect the Openfire news as part of the ignite realtime community

Reply:
Oh, I'm sorry. I didn't know that. This was my first time publishing a news about Openfire. I'll have to ask you what's the right way to do it for the next newsletter or we could fix it now if you want to :)